pkg/kv/kvserver/testdata/raft_log_truncator

```diff
@@ -385,7 +385,7 @@ r2.getStateLoader
 r2.setTruncatedStateAndSideEffects(..., expectedFirstIndex:32) => trusted:true
 r2.setTruncationDeltaAndTrusted(delta:-30, trusted:true)
 r2.setTruncatedStateAndSideEffects(..., expectedFirstIndex:33) => trusted:true
-r2.setTruncationDeltaAndTrusted(delta:-60, trusted:false)
+r2.setTruncationDeltaAndTrusted(delta:-50, trusted:false)
```

Comment:
This change from `-60` to `-50` is correct. There was a "bug" in how this test works. The `add-pending-truncation` in line 360 says `sideloaded-bytes=10 sideloaded-err=true`, and so the 10 bytes are now correctly not added to the stats.